src/debuggee.ts

```diff
@@ -34,6 +33,11 @@ export interface DebuggeeProperties {
   statusMessage?: StatusMessage;
 }
 
+export interface AppInfo {
```

Comment:
`PackageInfo` seems more accurate to me, since `AppInfo` implies that the package.json file being read is that of the application's.

Reply:
Done